core/client/src/test/java/alluxio/client/file/FileOutStreamTest.java

```diff
@@ -97,11 +104,13 @@
   @Before
   public void before() throws Exception {
     ClientTestUtils.setSmallBufferSizes();
+    mDelegationEnabled = Configuration.getBoolean(PropertyKey.USER_UFS_DELEGATION_ENABLED);
```

Comment:
Oh okay, if that's the intention I don't think its necessary, since we have broken up the tests which explicitly require it to be enabled or disabled.